packages/framework-extra-bundle/DependencyInjection/Integration/CronJobIntegration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Draw\Bundle\FrameworkExtraBundle\DependencyInjection\Integration;
+
+use Draw\Component\Console\Command\PurgeExecutionCommand;
+use Draw\Component\Console\Entity\Execution;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+class CronJobIntegration implements IntegrationInterface, PrependIntegrationInterface
+{
+    use IntegrationTrait;
+
+    public function getConfigSectionName(): string
+    {
+        return 'cron-job';
```

Comment:
use cron_job which is the convention of configuration name in symfony (underscore and not dash).

I'll adjust the maker command I tested with "demo" package name it's why I forgot that.